### ISSUE TITLE
Wrong indentation when using `method_docstring_numpy`.

### ISSUE THREAD

**Issue by yanghaoxie** [AndreaCrotti/yasnippet-snippets#349]
I am using Emacs 27.05. and I have
```
(setq python-indent-offset 4 
      python-indent-guess-indent-offset t)
```

- When I expand `method_docstring_numpy`, I get

```
    def foo(self, a, b, c):
        """
        
Parameters
----------
a : 

b : 

c : 


Returns
-------
out : 

        """
        
```
You see that the `Parameters` and `Returns` did not indent right.

- However, `method_docstring` expand right.
```
    def bar(self, a, b, c):
        """
        Keyword Arguments:
        a -- 
        b -- 
        c -- 
        """

```

**Comment by npostavs** [comment]
Works for me, I get
```
def foo(self, a, b, c):
    """
    
    Parameters
    ----------
    a : 

    b : 

    c : 


    Returns
    -------
    out : 

    """
    
```

**Comment by JCZuurmond** [comment]
Can anyone tell me which settings they have to fix this? My indentation is off for both cases, I guess I am missing some setting.

**Comment by AndreaCrotti** [comment]
Mm I just tried as well and it works for me, and my Emacs configuration is just https://github.com/AndreaCrotti/Emacs-configuration/blob/master/ca-init.el
@JCZuurmond 
Maybe you have some setting that affects this somehow?

**Comment by JCZuurmond** [comment]
@AndreaCrotti : Could you be a bit more precise in pointing out which configuration settings should effect this? I do not see any off:

```
(setq python-indent-offset 4 
      python-indent-guess-indent-offset t
```

**Comment by AndreaCrotti** [comment]
Yeah that was the point @JCZuurmond I don't have any custom config at all now since I don't really use Python.
So if you have a link to your config I guess it's probably something in your config that's affecting it.

**Comment by JCZuurmond** [comment]
You can find it [here](https://github.com/JCZuurmond/dotfiles/blob/master/emacs/.spacemacs). Thanks for helping, @AndreaCrotti!
